airflow/api/common/experimental/mark_tasks.py

```diff
@@ -187,7 +188,7 @@ def get_subdag_runs(dag, session, state, task_ids, commit, confirmed_dates):
                 dag_runs = _create_dagruns(current_task.subdag,
                                            execution_dates=confirmed_dates,
                                            state=State.RUNNING,
-                                           run_id_template=BackfillJob.ID_FORMAT_PREFIX)
+                                           run_id_template=DagRunType.BACKFILL_JOB.value + "{}")
```

Comment:
Can this method create this template by itself?  It seems to me that this template does not change and it will always be {DagRunType}__{execution_date}

Reply:
Nearly always, backfill has only one underscore

Reply:
We need to standardize it if we want to introduce a new column in the table later.  I would be happy if this PR cleaned up all the weirdnesses about DagRunType. In the next PR, we will update the database model and queries.

Cleaning up this mess is crucial for performance.
```
select * from test_runs where run_type=‘{}’
(0.0284, 0.037, 0.025)
select * from test_runs where run_id  like ‘{}%’
(0.0426, 0.0531, 0.0377)
type / like [-33.35%, -28.89%, -32.99%]
(avg, min, max)
```

Reply:
@mik-laj I agree so I used the double underscore